distributed/protocol/__init__.py

```diff
@@ -99,4 +100,5 @@ def _register_cudf():
 @dask_serialize.register_lazy("cuml")
 @dask_deserialize.register_lazy("cuml")
 def _register_cuml():
-    from cuml.comm import serialize
+    with suppress(ImportError):
```

Comment:
I think in dask this is typically written as 

```python
with ignoring(ImportError):
    ...
```

Reply:
Thanks! Fixed 😄

It would be interesting to know why that is. I would assume it is some holdover from when Python 2 was still supported. If so, would it make sense to move to the standard library equivalent? 😉

Reply:
> would it make sense to move to the standard library equivalent?

Yeah! xref https://github.com/dask/distributed/issues/3500